tests/test_imagereco_forward_model.py

```diff
@@ -0,0 +1,52 @@
+import os
+import tempfile
+
+import numpy as np
+from scipy.sparse import find
+
+import cedalion.datasets
+import cedalion.imagereco.forward_model as fw
+
+
+def allclose(A, B, atol=1e-8):
+    """Check if two sparse matrices are equal within a tolerance."""
+    # If you want to check matrix shapes as well
+    if np.array_equal(A.shape, B.shape) == 0:
+        return False
+    r1, c1, v1 = find(A)
+    r2, c2, v2 = find(B)
+    index_match = np.array_equal(r1, r2) & np.array_equal(c1, c2)
+    if index_match == 0:
+        return False
+    return np.allclose(v1, v2, atol=atol)
+
+
+def test_TwoSurfaceHeadModel():
+    ### tests only save and load methods so far
+    # prepare test head
+    (
+        SEG_DATADIR,
+        mask_files,
+        landmarks_file,
+    ) = cedalion.datasets.get_colin27_segmentation()
```

Comment:
The colin27 dataset is quite large and consequently the test is rather slow on my machine. We should add a smaller segmentation dataset for unit tests.

Reply:
Yes, true point. I'll try to create one